src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManager.java

```diff
@@ -567,8 +567,8 @@ protected JDAImpl buildInstance(final int shardId) throws LoginException
             }
             catch (CompletionException e)
             {
-                if (e.getCause() instanceof LoginException)
-                    throw (LoginException) e.getCause(); // complete() can't throw this because its a checked-exception so we have to unwrap it first
+                if (e.getCause() instanceof InvalidTokenException)
+                    throw (InvalidTokenException) e.getCause(); // complete() can't throw this because its a checked-exception so we have to unwrap it first
```

Comment:
I don't think we have to do this anymore because it isn't a checked exception

Reply:
Thank you for reviewing thus PR.
Do I need to remove the entire try catch or only the if block?